spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -589,8 +610,8 @@ function onTimeoutPacket(packet: Packet) {
 
 ```typescript
 function isSource(portId: string, channelId: string, token: Token) {
```

Comment:
Code comments explaining why would be good

Reply:
I rename the function and added a comment. Let me know what you think.